### DISCUSSION TITLE
Plus software PlusApp-2.8.0.20191105-Telemed-Win32.exe keeps failing to start

### DISCUSSION THREAD

**Discussion by MarionCrouzier** [PlusToolkit/PlusLib#811] [Q&A] [answered]
Hello everybody,
I am trying to set-up 3d slicer to be able to do real time 3d reconstruction from ultrasound (Telemed) and Optitrack (Trio). I have downloaded the Plus software (PlusApp-2.8.0.20191105-Telemed-Win32.exe), but I have troubles installing it on my computer. It actually seems to install the software but when I try to start the application, I get (again) the page to start installation from beginning (it is the screenshot attached). I have been installing VS2015 redistributable and still have the same problem. I am using Windows 10.
Anyone know what is going on?
Thank you!
Marion
![Plus problem](https://user-images.githubusercontent.com/84898062/119781353-357daf80-becb-11eb-91ce-08750576c4f2.PNG)

**Comment by Sunderlandkyl** [comment, marked as answer]
Are you starting PlusServerLauncher.exe?

**Reply by MarionCrouzier** [reply]
That was the good question that made me realize I did not...
Problem solved then, thank you!
